lib/mongo_conn.js

```diff
@@ -0,0 +1,60 @@
+/* eslint no-process-env: "off" */
+const { MongoClient } = require('mongodb');
+let hosts;
+let mongo;
+if (process.env.AMK_MONGO_HOSTS) {
+    const _hosts = [];
+    hosts = process.env.AMK_MONGO_HOSTS.split(',');
```

Comment:
please put all the process.env in another file. for example, `config/index`

Reply:
why need to put them in another file ? this component will be in node_modules folder,  is there any one who would update the config file in node_modules ?